rllib/algorithms/sac/torch/default_sac_torch_rl_module.py

```diff
@@ -37,13 +43,31 @@ def _forward_inference(self, batch: Dict) -> Dict[str, Any]:
         pi_encoder_outs = self.pi_encoder(batch)
 
         # Pi head.
-        output[Columns.ACTION_DIST_INPUTS] = self.pi(pi_encoder_outs[ENCODER_OUT])
+        if isinstance(self.action_space, gym.spaces.Discrete):
+            # For discrete action spaces, we return the logits for each action.
+            output[Columns.ACTIONS] = self.pi(pi_encoder_outs[ENCODER_OUT])
```

Comment:
Aren't these also `ACTION_DIST_INPUTS`?
We use these (same as for cont. actions) to create a distribution from these and then sample from that distribution.